charters/DataOperations/charter.md

```diff
@@ -0,0 +1,101 @@
+  - Ensure proper checks of data quality standards are put in place within the data workflow
+  - Engage with the [Compliance Working Group](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Compliance-WG/charter.md) to ensure the our data release and distribution processes ensure responsible HCA data sharing to all types of consumers
+  - Ensure released data that are used on tertiary portals can be easily cross-reference across portals
+  - Determine best practices for when and how changes are made to DCP outputs after release
+
+- Community engagement
+  - Define data use policy and citation instructions for HCA data users
+  - Define mechanism(s) for which the community is notified when DCP products are updated after being released
+  - Partner with Content and Portal Development to ensure clear, findable, and current documentation describing the data workflow and data distribution
+  - Ensure effective communication of HCA data updates and user features to the scientific community
+  - Partner with UX to assess curation added value of data distribution through communities’ feedback, and better understand their expectations and needs
+  - Onboard staff to DCP ZenDesk ticketing system
+  - Take ownership of establishing process and interfacing with DCP components for assigning, responding, and integrating ZenDesk tickets
+  - Compile feedback gained from consumers through ZenDesk queries and comments, and report back to the DCP in order to facilitate system improvement 
+
+
+### Scientific "guardrails"
+
+In order to meet the needs of the communities of scientific consumers, the DataOps team requires input from the Governance Group as to where curated data distribution will add value for the researcher.
+
+## Out-of-scope
+
+The Data Operations is not responsible for determining what data or data types are made available through the DCP.
+
+As many of the features and requirements for the data workflow and data distribution will be implemented through other components, the majority of technical aspects pertaining to the data workflow and data distribution will be implemented by the appropriate component, and not Data Operations.
+
+## Milestones and Deliverables
+
+2019 Q3
+- Follow a dataset through the entire data workflow in order to understand each step and assess the current process. 
+- Organize incentivized data contribution program in conjunction with the HCA General Meeting in Barcelona in order to encourage pre-publication data contribution. 
+- Establish ZenDesk ticket triage procedure.
+
+2019 Q4
+- Write actionable RFCs aimed to improve the data workflow.
+- Establish an MVP of features and improvements that are the prerequisite to a first Data Distribution
+
+2020 Q1
+- Determine a reproducible procedure for a periodic data snapshot including:
```

Comment:
I would also be interested in seeing thought towards how the cadence of distribution is defined. I do not believe that timed intervals are the best way of performing distribution and would encourage including metrics that require thinking of the user value being delivered when motivating distribution events.

Reply:
We see use cases for timed interval distributions, termed Snapshots, but not all distributions are on timed intervals. We certainly intend to listen to the community and respond accordingly. We tried to capture that with this objective: "Partner with UX to assess the added value of data distribution through communities’ feedback, and better understand their expectations and needs". Are you looking for more specifics?

Reply:
@TimothyTickle is there still concern over our general distribution plans?

Reply:
yes, time being one factor of many makes sense.